addons/godot_pixelorama_importer/pixelorama_file_open/inspector_menu.tscn

```diff
@@ -0,0 +1,51 @@
+[gd_scene load_steps=2 format=2]
+
+[ext_resource path="res://addons/godot_pixelorama_importer/pixelorama_file_open/configure_dialog.tscn" type="PackedScene" id=1]
+
+[node name="Control" type="Control"]
+anchor_right = 1.0
+anchor_bottom = 1.0
+size_flags_horizontal = 3
+size_flags_vertical = 9
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="VBoxContainer" type="VBoxContainer" parent="."]
+anchor_right = 1.0
+anchor_bottom = 1.0
+margin_left = 0.629944
+margin_top = 5.62996
+margin_right = 0.629883
+margin_bottom = 0.629944
+size_flags_horizontal = 3
+size_flags_vertical = 9
+custom_constants/separation = 10
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="Label" type="Label" parent="VBoxContainer"]
+margin_right = 1023.0
+margin_bottom = 30.0
+rect_min_size = Vector2( 0, 30 )
+text = "Pixelorama Plugin"
+valign = 1
+
+[node name="OpenButton" type="Button" parent="VBoxContainer"]
+margin_top = 40.0
+margin_right = 1023.0
+margin_bottom = 70.0
+rect_min_size = Vector2( 0, 30 )
+text = "Open in Pixelorama"
```

Comment:
For some reason this button doesn't update right after I set the path. Could you have a look on this? :thinking:

Reply:
That's by design, you just need to re-select the image node and it will reappear. I did not want write some complex logic just to get that working.

Reply:
Ah alright, I guess that'll do for now :P